mssqlcli/localized_strings.py

```diff
@@ -0,0 +1,20 @@
+# pylint: skip-file
+import gettext
+import os
+from importlib import import_module
+
+PATH = '{0}.py'.format(os.path.splitext(__file__)[0])
+DOMAIN = 'mssql-cli'
+LOCALE_DIR = os.path.join(os.path.dirname(__file__), 'locale')
+LANGUAGES = None
+
+def translation(domain=DOMAIN, localedir=LOCALE_DIR, languages=None):
```

Comment:
Can we validate the language items such that we throw if passed and invalid language?

Reply:
will tackle in separate PR as we discussed offline.